kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKmsCloud.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kms.aws;
+
+import java.net.URI;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.systemtests.Environment;
+import io.kroxylicious.systemtests.executor.Exec;
+import io.kroxylicious.systemtests.executor.ExecResult;
+
+/**
+ * The type Aws kms cloud.
+ */
+public class AwsKmsCloud implements AwsKmsClient {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsKmsCloud.class);
+    private static final String AWS_CMD = "aws";
+    private String region;
+
+    /**
+     * Instantiates a new Aws.
+     *
+     */
+    public AwsKmsCloud() {
+        // nothing to initialize
+    }
+
+    @Override
+    public String getAwsCmd() {
+        return AWS_CMD;
+    }
+
+    @Override
+    public boolean isAvailable() {
+        boolean awsCloudSelected = Environment.AWS_USE_CLOUD.equalsIgnoreCase("true");
+        boolean keyIdDefaulted = Environment.AWS_ACCESS_KEY_ID.equals(Environment.AWS_ACCESS_KEY_ID_DEFAULT);
+        if (awsCloudSelected && !keyIdDefaulted) {
+            LOGGER.atInfo().log("Using AWS Kms Cloud");
+        }
+        else {
+            if (awsCloudSelected) {
+                LOGGER.atWarn().log("AWS Cloud selected, but AWS_ACCESS_KEY_ID remains defaulted. Please insert a correct key id.");
+            }
+            else if (!keyIdDefaulted) {
+                LOGGER.atWarn().log("AWS LocalStack selected, but AWS_ACCESS_KEY_ID is not defaulted. Please insert a correct key id or select AWS Cloud use.");
```

Comment:
grammar nit:

```suggestion
                LOGGER.atWarn().log("AWS LocalStack selected, but AWS_ACCESS_KEY_ID is not specified. Please specify a key id or set AWS_USE_CLOUD use.");
```

Though actually we should reference the actual environment vars here too so it should be `Environement.AWS_ACCESS_KEY_ID_VAR` (I think)